dependencies/graknlabs/dependencies.bzl

```diff
@@ -29,12 +29,12 @@ def graknlabs_grakn_core():
     git_repository(
         name = "graknlabs_grakn_core",
         remote = "https://github.com/graknlabs/grakn",
-        tag = "1.7.2", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
+        commit = "8aa8721caf9a83ed7651e2612686f7738cd323e7", # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_grakn_core
```

Comment:
this is now released! 1.8.0

Reply:
addressed in https://github.com/graknlabs/workbase/pull/298/commits/5e7cb8a458fabb8f8106071aa7ca1417e38417c3